packages/SwingSet/src/kernel/deviceSlots.js

```diff
@@ -132,9 +132,10 @@ export function makeDeviceSlots(
       throw new Error('SO(SO(x)) is invalid');
```

Comment:
this could be changed to `assert(!outstandingProxies.has(x), 'SO(SO(x)) is invalid')`, but I don't know if you're aiming to change `throw`s that don't use `details`

Reply:
I am. Thanks.